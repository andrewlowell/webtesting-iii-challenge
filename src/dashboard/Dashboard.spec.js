import React from 'react';
import renderer from 'react-test-renderer';
import { render, fireEvent } from "react-testing-library";
import "jest-dom/extend-expect";

import Dashboard from './Dashboard';

describe('<Dashboard />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Dashboard />).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
