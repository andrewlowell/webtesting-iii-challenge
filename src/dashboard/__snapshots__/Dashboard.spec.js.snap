// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> should match snapshot 1`] = `
Array [
  <div
    className="display panel"
  >
    <div
      className="led green-led"
      data-testid="disp-lock"
    >
      Unlocked
    </div>
    <div
      className="led green-led"
      data-testid="disp-open"
    >
      Open
    </div>
  </div>,
  <div
    className="controls panel"
  >
    <button
      className="toggle-btn"
      data-testid="tog-lock"
      disabled={true}
      onClick={[Function]}
    >
      Lock Gate
    </button>
    <button
      className="toggle-btn"
      data-testid="tog-open"
      disabled={false}
      onClick={[Function]}
    >
      Close Gate
    </button>
  </div>,
]
`;
